flapp-webapp/src/styles/styles.scss

```diff
@@ -1,6 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
-$bcgov-font-path: "../node_modules/@bcgov/bootstrap-theme/dist/fonts/";
-@import "~@bcgov/bootstrap-theme/dist/scss/bootstrap-theme";
+//$bcgov-font-path: "../node_modules/@bcgov/bootstrap-theme/dist/fonts/";
```

Comment:
Why change this to a CSS import instead of SCSS? Now the theme variables can't be customized.